town.logic

```diff
@@ -1,2 +1,107 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Anju:
+      chest AnjuHP
+    node AnjuRoof:
+      door AnjuChimney
+    node CatsMinish:
+      door Fountain
+      door StratoMinishBack
+    node FigurineHouseDoorEast:
+      door FigurineHouseEast
+    node GormanHouseSouth:
+      door GormanHouseSouth
+    node HearthLedge:
+      door HearthLedge
+      chest HearthLedge
+    node JuliettaRoof:
+      door JuliettaChimney
+      door RomioChimneyEast
+    node LibraryRoof:
+      door LibraryRoof
+    node LibraryRoofMinish:
+      door FlipperCave
+      door LibraryMinish
+    LibraryRoof -> LibraryRoofMinish: item Cane
+    node Main:
+      chest CafeLady
+      door Bakery
+      door Borlov
+      door East
+      door FigurineHouseWest
+      door HearthNorth
+      door HearthSouth
+      door GormanHouseWest
+      door Julietta
+      door LibraryBottom
```

Comment:
This door needs the old man to move to be accessible - at least for the entrance side (exiting without that flag set needs testing).

Reply:
I just tested, it works just fine to exit with the man standing there, so ill go ahead and update it

Reply:
So this should be just the exit (with the entrance elsewhere)
```suggestion
      exit LibraryBottom
```